### PR TITLE
[Contributing.md] Suggestion to improve the contribution best practices

### PR DESCRIPTION
**Pull request by cwilso**
This is merely a suggestion to improve the contribution best practices to reflect (what I think is) the actual status quo.

This is based on an attempt to figure out how we could have avoided some incidents that ruffled a few feathers, by being clearer about expectations;  I'd welcome feedback about whether this is useful or if it could be improved in some other way.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (fantasai) [inline CONTRIBUTING.md:55]: Non-normative text doesn't generally require WG consensus, only if it's reflecting something sensitive. Substantial changes to non-normative text should have the editors approval though.

File: CONTRIBUTING.md
Comment (fantasai) [inline CONTRIBUTING.md:61]: s/should/must/

File: CONTRIBUTING.md
Comment (fantasai) [inline CONTRIBUTING.md:64]: s/replacement/substitute/

File: CONTRIBUTING.md
Comment (cwilso) [reply CONTRIBUTING.md]: Addressed.

File: CONTRIBUTING.md
Comment (cwilso) [reply CONTRIBUTING.md]: Changed.

**Review by frivoal — APPROVED** [review]

**Review by hober — APPROVED** [review]
Looks good to me!

**Comment by cwilso** [comment]
As this isn't a spec, unfortunately these guidelines aren't quite applicable (since there's no Editor :P ).  As I note a couple of approvals and a thumbs-up from one of the chairs, I'll leave it to you all to decide when it's appropriate to merge once @fantasai has given it a re-review.

**Review by tantek — APPROVED** [review]
LGTM, this is consistent with my rough understanding of current WG expectations.

**Comment by astearns** [comment]
We will announce this update on the Feb 28 call and can take further improvements

**Comment by bramus** [comment]
To those following: filed a small PR to clarify where PRs should originate from: https://github.com/w3c/csswg-drafts/pull/10009

**Pull request merged**
